README.md

```diff
@@ -85,17 +77,17 @@ Legend:
 ✅ quay.io/prometheus/prometheus:v2.42.0
 ✅ redis:6.2.4-alpine
 ✅ registry.k8s.io/autoscaling/cluster-autoscaler:v1.25.3
-🚫 registry.k8s.io/ingress-nginx/controller:v1.9.4@sha256:5b161f051d017e55d358435f295f5e9a297e66158f136321d9b04520ec6c48a3
 ✅ registry.k8s.io/kube-state-metrics/kube-state-metrics:v2.8.1
 ```
 
 ### Errors in Output
 
-At present the tool will show an error if it cannot connect to the repository for an image for any reason.
-This could include:
-* Authentication failure
-* Communication error
-* Image no longer available
+If there is an error whilst checking an image, the tool will display the 🚫 symbol and give a short description of the error at the end of the line. The current common errors are:
+
+- Authentication error. This usually means that you are missing your docker credentials or that you are denied access to the registry.
+- Communication error. Could mean that the URL host doesn't exist or that its down.
+- Image not found.
```

Comment:
Perhaps add some explanation, the image is listed in xxxx but could not be found in yyyy

Reply:
I have now made a change that might address this, but I could be wrong,  can you please elaborate on what you mean?

Reply:
I also feel like the readme was a bit misleading, the errors are meant to be short so they only say:
- authentication error
- communication error
- image not found, some pods like xx are using images that no longer exist.

I have updated the readme to reflect this.